build_img.sh

```diff
@@ -141,6 +141,8 @@ function create_image () {
     dd if=output/imx-boot-sd.bin of=${PARTITION}1 bs=1K status=noxfer status=progress 2>/dev/null
 
     echo "Installing OS                 to ${PARTITION}3"
+    #pv output/rootfs-base.tar.gz | tar -xzp -C sdcard
```

Comment:
If you think is often needed please put a switch to the script or removed the commented out code (I know, I should do the same)